src/GitHub.Extensions/TaskExtensions.cs

```diff
@@ -20,5 +20,8 @@ public static class TaskExtensions
                 return default(T);
             }
         }
+        public static void Forget(this Task task)
```

Comment:
Huh? What's this for? Shouldn't we just call `task.Wait` where we'd call this method?